src/maud/sampling.py

```diff
@@ -74,6 +75,53 @@ def get_full_stoichiometry(
     return S_enz, S_enz_to_flux_map, S_complete, S_drain
 
 
+def validate_specified_fluxes(
+    S_complete,
+    rxn_measurements,
+    experiment_codes: Dict[str, int],
+    reaction_codes: Dict[str, int],
+    drain_codes: Dict[str, int],
+    balanced_mic_codes: Dict[str, int],
+):
+    """Validate measured fluxes.
```

Comment:
Could make this description a bit more specific - maybe "Check that appropriate fluxes have been measured."